.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "wpt"]
+	path = wpt
+	url = git@github.com:web-platform-tests/wpt
```

Comment:
Instead of using a git submodule here, it'd be great if we could use CMake/CPM to check out the suite. I left a suggested edit in `wpt.cmake` for how.